### PR TITLE
generated new imcsdk and updated mo, meta and respected files

### PR DESCRIPTION
**Pull request by PavanMuchhukunta**

### CODE REVIEW

**Comment by PavanMuchhukunta** [comment]
"make test"

File: tests/unit_tests/common/test_imcversion.py [hunk 1]
```diff
@@ -14,6 +14,17 @@
 from nose.tools import assert_equal
 from imcsdk.imcmeta import VersionMeta
 from imcsdk.imccoremeta import ImcVersion
+from ..connection.info import custom_setup, custom_teardown
+
+
+def setup_module():
```
Comment (vvb) [inline tests/unit_tests/common/test_imcversion.py]: setup_module and teardown_module must not be used in this test. 
Everything under tests/unit_tests is expected to work without a connection to an actual server. 

If any of your tests need the handle.login(), then you need to move them out of unit_tests

File: tests/unit_tests/test_ipmi.py [hunk 2]
```diff
@@ -11,7 +11,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from mock import patch, MagicMock
+try:
```
Comment (vvb) [inline tests/unit_tests/test_ipmi.py]: if unittest.mock provides the same functionality, then let us use that and remove the `from mock import patch, MagicMock` all together. No need to do try and except.

File: tests/unit_tests/common/test_imcversion.py (hunk 1 above)
Comment (PavanMuchhukunta) [reply tests/unit_tests/common/test_imcversion.py]: sure vikrant, i will update the code

File: tests/unit_tests/test_ipmi.py (hunk 2 above)
Comment (PavanMuchhukunta) [reply tests/unit_tests/test_ipmi.py]: it's already removed vikrant, now i don changes only for "test_imcversion.py"

**Review by vvb — APPROVED** [review]

**Pull request merged**